sdk/monitor/microsoft-opentelemetry-exporter-azuremonitor/microsoft/opentelemetry/exporter/azuremonitor/export/__init__.py

```diff
@@ -62,7 +60,8 @@ def _transmit_from_storage(self) -> None:
             # give a few more seconds for blob lease operation
             # to reduce the chance of race (for perf consideration)
             if blob.lease(self._timeout + 5):
-                envelopes = blob.get()
+                envelopes = map(
+                    lambda x: TelemetryItem(**x), blob.get())
```

Comment:
I'm not familiar with `TelemetryItem(**x)` syntax. How does it work?

Reply:
Basically sending the constructor parameters as keyword arguments so we only send the values that exist in "x" to avoid unnecessary None parameters

Reply:
I think also during the archboard review yesterday, someone said something about using list comprehension instead of `map`?

Reply:
Updated the code to reflect that one